src/sync-controller.js

```diff
@@ -6,6 +6,10 @@ import {sumDurations, getPartsAndSegments} from './playlist';
 import videojs from 'video.js';
 import logger from './util/logger';
 
+// The maximum gap allowed between two media sequence tags when trying to
+// synchronize expired playlist segments.
+const MAX_MEDIA_SEQUENCE_DIFF_FOR_SYNC = 1000;
```

Comment:
@brandonocasey do you think it would be worth having this be in config.js or just have it be here and if anyone complains we can move it to config.js?

Reply:
I think we can just leave it here. I highly doubt that we will need to have it configured.

Reply:
Have accepted the suggested message change, and added a unit test. Let me know if there's anything else I can do.

Reply:
```suggestion
// the max media sequence diff is 48 hours of live stream 
// content with two second segments. Anything larger than that
// will likely be invalid.
const MAX_MEDIA_SEQUENCE_DIFF_FOR_SYNC = 86400;
```

One of our colleagues mentioned that 1000 might not be big enough for long running live streams. We all agreed on a number that we were happy with. Sorry about that.